crates/node/src/lib.rs

```diff
@@ -590,24 +590,24 @@ async fn provision_cluster_metadata(
     .await?;
 
     retry_on_network_error(common_opts.network_error_retry_policy.clone(), || {
-        write_initial_value_dont_fail_if_it_exists(
+        write_initial_logs_dont_fail_if_it_exists(
             metadata_store_client,
-            PARTITION_TABLE_KEY.clone(),
-            &initial_partition_table,
+            BIFROST_CONFIG_KEY.clone(),
+            initial_logs.clone(),
         )
     })
     .await
-    .context("failed provisioning the initial partition table")?;
+    .context("failed provisioning the initial logs")?;
 
     retry_on_network_error(common_opts.network_error_retry_policy.clone(), || {
         write_initial_value_dont_fail_if_it_exists(
             metadata_store_client,
-            BIFROST_CONFIG_KEY.clone(),
-            &initial_logs,
+            PARTITION_TABLE_KEY.clone(),
+            &initial_partition_table,
         )
     })
     .await
-    .context("failed provisioning the initial logs")?;
+    .context("failed provisioning the initial partition table")?;
```

Comment:
Maybe add a comment explaining why it is important that we only create the `PartitionTable` after we have created the `Logs` configuration so that I won't change it in the future accidentally.

Reply:
Done :)